libsolidity/inlineasm/AsmParser.cpp

```diff
@@ -606,7 +606,7 @@ bool Parser::isValidNumberLiteral(string const& _literal)
 {
 	try
 	{
-		u256(_literal);
+		u256 _literal;
```

Comment:
This is actually a bug in Solidity code where they seem to want to do `(void)u256(_literal)` instead of decelerating a local that shadows the parameter for nothing.